telegram/_message.py

```diff
@@ -198,7 +198,7 @@ def _de_json(
             data["date"] = from_timestamp(data["date"], tzinfo=loc_tzinfo)
 
         data["chat"] = Chat.de_json(data.get("chat"), bot)
-        return super()._de_json(data=data, bot=bot)
+        return super()._de_json(data=data, bot=bot, api_kwargs=api_kwargs)
```

Comment:
wtf, how did we miss this so far? why don't the tests complain about that? o.0

Reply:
no idea, I was pretty surprised too